src/components/tools/WindowLevelTool.vue

```diff
@@ -154,12 +154,29 @@ const WindowLevelToolComponent = defineComponent({
       );
     }
 
+    let lastValues = '';
     watch(
       () => [wwRange.value, wlRange.value],
       ([ww, wl]) => {
-        vertVal.value = ww.default;
-        horizVal.value = wl.default;
-        updateManipulator();
+        // Avoid finicky vertVal update by only calling updateManipulator
+        // if actual values changed, not just object reference.
+        // Window config object is updated with every change to window/level.
```

Comment:
Good catch. Can we just add `deep: true` to the watch options to get this to work instead of tracking changes ourselves?

Reply:
Refactored to make `vertVal` a computed ref, then the manipulator "set" function sets the Pinia store `width`.